home-manager/ssh.nix

```diff
@@ -69,21 +70,31 @@ in
     # Enable custom or temporary config without `home-manager switch`
     includes = [ "${sshDir}/config.local" ];
 
+    # Workaround for GH-950
+    extraOptionOverrides = {
+      # https://github.com/sul-dlss/DeveloperPlaybook/blob/32b384111f7f6a79ec21f305cf4bdd10e8292f5c/best-practices/ssh_configuration.md?plain=1#L14-L15
+      IgnoreUnknown = "GSSAPI*";
```

Comment:
https://github.com/openssh/libopenssh/blob/05dfdd5f54d9a1bae5544141a7ee65baa3313ecd/ssh/servconf.c#L355C1-L360

Might be different for the unknown and unsupported? :thinking: 

```
#ifdef GSSAPI
	{ "gssapiauthentication", sGssAuthentication, SSHCFG_ALL },
	{ "gssapicleanupcredentials", sGssCleanupCreds, SSHCFG_GLOBAL },
```